src/matching/quoteMatcher.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * QuoteMatcher matches quoted strings, respecting escaped quotes (\") and frieds
+ */
+export class QuoteMatcher {
+  static escapeChar = "\\";
+
+  static findOpening(char: string, corpus: string, start: number): number {
+    // first, search backwards to see if we could be inside a quote
```

Comment:
Great! This doesn't surprise me as Vim is not always consistent with its documentation. As it's kind of `incomplete`(Vim's behavior), do you mind adding this `"foo"‹cursor›"bar"` case to test suite then other people won't see it as our code issue?